src/smrt/shift_heat_fluids_driver.cpp

```diff
@@ -20,9 +20,7 @@
 #include <gsl/gsl>
 
 // enrico includes
-#ifdef USE_SHIFT
```

Comment:
You've removed this and the other `#ifdef` because they're already handled by the `if (SCALE_FOUND)` conditionals in `CMakeLists.txt`, correct?

Reply:
Yes, that's correct. And it looks like `USE_SHIFT` was never actually set even when `SCALE_FOUND` was true.

Reply:
`USE_SHIFT` is actually a macro defined by SCALE identifying whether Shift is enabled as a package.  We should force Shift to be enabled by the SCALE `find_package`.

Reply:
Ah, ok. What would you like me to do on this branch then?

Reply:
If we remove `USE_SHIFT`, then we should probably include Shift in the package list in the `find_package(SCALE ...)` call in `CMakeLists.txt`.  I don't think there is a viable reason to have SCALE around without Shift.  I believe my original idea was that some of my surrogate stuff used lower level SCALE components so you could plausibly use the surrogates even if Shift wasn't enabled.  I don't think there's a real reason to support that use case.

Reply:
:+1: Just added Shift to the list of components